test-case/run-all-tests.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 set -e
 
+# This entire script is a hack. We need to re-use some existing test
+# runner from some existing test framework eventually
+
+very_short_tests()
+{
+	:  "${small_loop:=1}"
+	:  "${medium_loop:=1}"
+	:  "${large_loop:=1}"
+
+	:  "${medium_count:=5}"
+	:  "${large_count:=10}"
+
+	: "${long_duration:=3}"
+
+	: "${large_round:=3}"
+}
+# very_short_tests
```

Comment:
could we make this configurable with e.g. a -s option?

Reply:
Done: `-l 1`